rusk-wallet/src/wallet/transaction.rs

```diff
@@ -488,11 +532,8 @@ impl<F: SecureWalletFile + Debug> Wallet<F> {
 
         let stake_pk = BlsPublicKey::from(&stake_sk);
 
-        let reward_amount = state
-            .fetch_stake(&stake_pk)
-            .await?
-            .map(|s| s.reward)
-            .unwrap_or(0);
```

Comment:
weird that it doesn't suggest to use `unwrap_or_default()`

Reply:
true! I didn't even see it.. Will change